app/controllers/auctions_controller.rb

```diff
@@ -1,13 +1,13 @@
 class AuctionsController < ApplicationController
   def index
     collection = AuctionQuery.new.public_index
-    @view_model = AuctionsIndexViewModel.new(current_user, collection)
+    @auctions = AuctionsIndexViewModel.new(auctions: collection, current_user: current_user)
```

Comment:
yes we talked about that! One benefit of having it be `current_user` is that the use case is super duper clear, but it's many more chars and I do love typing less. Hmm...